config/locales/alchemy.ru.yml

```diff
@@ -510,7 +510,7 @@ ru:
     "please enter subject and mail address": "Пожалуйста, заполните поля 'тема' и 'получатель'."
     please_confirm: "Пожалуйста, подтвердите"
     please_wait: "Подождите..."
-    position_in_text: "Положение в тексте"
+    position_in_text: "Положение в тек��те"
```

Comment:
We need to be careful with the russian translation: �
I saw this few days ago already.